Basic-Car-Maintenance/Shared/Dashboard/Views/AddMaintenanceView.swift

```diff
@@ -34,9 +34,9 @@ struct AddMaintenanceView: View {
                 }
                 
                 Section {
-                    Picker(selection: $selectedVehicle) {
-                        ForEach(vehicles) { vehicle in
-                            Text(vehicle.name).tag(vehicle as Vehicle)
+                    Picker(selection: $selectedVehicleID) {
+                        ForEach(vehicles, id: \.self) { vehicle in
```

Comment:
```suggestion
                        ForEach(vehicles) { vehicle in
```

This is not needed, it's about if the elements in the `ForEach` are `Identifiable`, which they are, the `id` doesn't need to be explicitly noted